src/sora-v3.md

```diff
@@ -0,0 +1,116 @@
+
+#### Increasing Output While Deflating XOR
+
+There are two principles of economics to keep in mind when creating an economic system:
+
+1. the principle of economic growth (ΔMV=PΔY)
+2. the principle of elasticity
+
+Regarding the first, you need to increase purchasing power for economic expansion. Otherwise, there will be crowding out of uses for existing money. Consider a simple example of a small economy with just three people, each having 2 XOR. In this simple economy, there are only 6 XOR, and if you want to build, let’s say, a bridge that costs 2 XOR to build, then that 2 XOR has to come from among the three people in the economy. So, to build the bridge without expanding the purchasing power in the economy, you need to convince people to give up their XOR, potentially promising some large bonus in the future (security).
+
+However, this can be a costly way to obtain funds if a very large bonus has to be paid to convince people to give up their XOR. So another option is to inject more XOR into the economy to build the bridge but then charge some fees to use the bridge to withdraw and burn that XOR later in the future. In addition to lowering the cost of infrastructure development, this simple example also highlights the principle of how the elasticity of purchasing power can be used to provide for production without crowding out other uses of money in an economy, as the original three people in the example can use their funds for different uses, like investing in opportunities that will generate even more wealth in the economy.
+
+#### The Vision for SORA V3
+
+We can build a sustainable and self-reinforcing economy by using buy-back-and-burn mechanisms (a portion of the VAL and PSWAP re-mint to go to burning KUSD, while still keeping VAL and PSWAP strictly deflationary, as well as a portion of the XOR network fees paid with each transaction) to help KUSD maintain its peg, with a large TBCD treasury to cover any gaps, with an additional mechanism to buy-back-and-burn TBCD (10% of the remint of VAL and PSWAP—which captures both the use and volume of activity on the network). Through this process, we can create a powerful vortex that will eventually lead to the burning of a large amount of the XOR supply while still creating conditions for financing new output in the SORA economy. Yes, it’s a long-term play, but creating a new world economy that isn’t based on debt and enslavement will take time and have a never-ending share of sceptics and critics.
```

Comment:
We should add a visual here to illustrate the scheme. Similar to the one in https://wiki.sora.org/kusd.html#support-kusd

Reply:
Done